recipes-kernel/linux/files/5.x-kernel/0003-setting-the-RJ45-port-led-behavior.patch

```diff
@@ -1,18 +1,18 @@
-From bd8c62c33aca5c94bc54c79fdef7c6c96ef57671 Mon Sep 17 00:00:00 2001
+From 025fdf9f8e0535d5156db70e5a7d1541df3ef4c8 Mon Sep 17 00:00:00 2001
 From: zengchao <chao.zeng@siemens.com>
 Date: Wed, 6 Nov 2019 11:21:49 +0800
-Subject: [PATCH 08/14] setting the RJ45 port led behavior
+Subject: [PATCH 03/14] setting the RJ45 port led behavior
```

Comment:
This patch is not needed. Firmware sets up related PHY register, Linux never touches it.

Reply:
Not yet. linux side would re-init the phy register. so if not the set the register on linux. the RJ45 Led behavior is not correct,

Reply:
I've tested it with the upstream firmware, and it was correct, though that was some time ago. Anyway, please point to the location where Linux is rewriting the register that U-Boot set up. That is needed as prove for such patch, specifically for upstream.

Reply:
Understood now why you found this still needed: We do not have prueth support in PG2 U-Boot, thus also no execution of board_phy_init. That leaves the phy in default configuration on PG2 (and only there). Let's see if we can get U-Boot support prueth for PG2 as well.

Reply:
IIRC，for our PG1 based on kernel 4.19.  If no this patch, RJ45 behavior is not correct